### DISCUSSION TITLE
Changing the Net target of the IDL generator

### DISCUSSION THREAD

**Discussion by mellegge** [jmmorato/openddsharp#170] [Q&A] [answered]
I followed the Getting Started page and followed the instructions to create the TestMessage project:

dotnet new openddsharp-idl-project --name TestMessage --output TestMessage

The output from this creates a project which is .Net Standard 2.0 project.  Is it possible to create an openddsharp-idl-project project targeting .Net 7.0?  Or does it really matter - can a .Net 7.0 project just pull in the the .NetStandard IDL project without issues?

Thanks
Mel
(Newbie to OpenDDSharp)

**Comment by jmmorato** [comment, marked as answer]
Both options are valid.

.Net Standard 2.0 libraries can be loaded in Net7.0 applications or you could change the IDL library project to use net7.0 if you are sure no other older version is going to be used.
